RELEASE/scripts/autoscend/auto_pre_adv.ash

```diff
@@ -347,6 +347,12 @@ boolean auto_pre_adventure()
 	{
 		autoEquip($slot[acc3], $item[backup camera]);
 	}
+
+	if(get_property("auto_forceNonCombatSource") == "jurassic parka" && !get_property("auto_parkaSpikesDeployed").to_boolean())
+	{
+		autoEquip($item[jurassic parka]);
+		auto_configureParka("spikes");
```

Comment:
```suggestion
		autoForceEquip($item[jurassic parka]); //equips parka and forbids maximizer tampering with shirt slot
		set_property("auto_parkaSetting","spikes"); //not using auto_configureParka("spikes") so maximizer stays aware of ML from shirt, instead of maximizing with another shirt or no shirt before changing to parka
```
not tested but could be better for the little ML control autoscend is capable of, though maybe auto_configureParka() could do it like this itself

Reply:
Good call. I'll commit it and test today

Reply:
Add a couple extra lines to ensure parka is in correct mode as well